CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+### Before you Start
+- Unless stated in README.md. Make sure you are pulling the source code from main branch with recursive variable to pull all submodules Collapse need to run.

```

Comment:
- Unless stated -> Unless otherwise stated...
- Remove the period after md, since it's a continuous sentence
- Make sure you are pulling -> make sure to clone from the main branch...
- with recursive variable -> using the `--recurse` parameter while cloning...
- all submodules -> all the submodules...

Final sentence: Unless otherwise stated in `README.md`, make sure to clone from the main branch using the `--recurse` option to pull all the submodules Collapse needs to run.

Reply:
Applied